src/main/java/com/exasol/dbbuilder/dialects/mysql/MySqlGlobalPrivilege.java

```diff
@@ -0,0 +1,11 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import com.exasol.dbbuilder.dialects.GlobalPrivilege;
+
+/**
+ * MySQL global privilege as used in a {@code GRANT}.
```

Comment:
Please add `@see` and a URL to the documentation. Also in the other files.

Reply:
Done